Tests/Unit/Controller/Backend/ApplicationControllerTest.php

```diff
@@ -180,9 +180,10 @@ public function listsMyApplications()
      */
     public function showsASingleApplication()
     {
+        $this->view->assign("print", 1)->shouldBeCalled();
```

Comment:
Can you update this too to use boolean values? Otherwise this will hit us once we use scalar type hints and strict types.